### PR TITLE
Add ASTAR's shibuya testnet

### PR DESCRIPTION
**Pull request by msantamaria86**
## Description
Added shibuya, a testnet for the Astar network on Polkadot (evm compatible)

## Additional Information
- [ X] I read the contributing docs

Your ENS/address:
0x0C5981f7091aa22C57C5871F5540e136a63bfF1D

### CODE REVIEW

File: packages/chains/src/shibuya.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    symbol: 'SBY',
+  },
+  rpcUrls: {
+    default: { http: ['https://evm.shibuya.astar.network/'] },
+    public: { http: ['https://evm.shibuya.astar.network/'] },
+  },
+  blockExplorers: {
+    default: {
+      name: 'Shibuya Explorer',
+      url: 'https://shibuya.subscan.io/',
```
Comment (tmm) [inline packages/chains/src/shibuya.ts:19]:
```suggestion
      url: 'https://shibuya.subscan.io',
```

File: packages/chains/src/shibuya.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  name: 'Shibuya Network',
+  network: 'Shibuya-network',
+  nativeCurrency: {
+    decimals: 18,
+    name: 'shibuya',
+    symbol: 'SBY',
+  },
+  rpcUrls: {
+    default: { http: ['https://evm.shibuya.astar.network/'] },
+    public: { http: ['https://evm.shibuya.astar.network/'] },
```
Comment (tmm) [inline packages/chains/src/shibuya.ts:14]:
```suggestion
    default: { http: ['https://evm.shibuya.astar.network'] },
    public: { http: ['https://evm.shibuya.astar.network'] },
```

File: packages/chains/src/shibuya.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  blockExplorers: {
+    default: {
+      name: 'Shibuya Explorer',
+      url: 'https://shibuya.subscan.io/',
+    },
+  },
+  testnet: true,
+}
```
Comment (tmm) [inline packages/chains/src/shibuya.ts:23]:
```suggestion
export const shibuya = {
  id: 81,
  name: 'Shibuya Network',
  network: 'Shibuya-network',
  nativeCurrency: {
    decimals: 18,
    name: 'shibuya',
    symbol: 'SBY',
  },
  rpcUrls: {
    default: { http: ['https://evm.shibuya.astar.network/'] },
    public: { http: ['https://evm.shibuya.astar.network/'] },
  },
  blockExplorers: {
    default: {
      name: 'Shibuya Explorer',
      url: 'https://shibuya.subscan.io/',
    },
  },
  testnet: true,
} as const satisfies Chain
```

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a patch [changeset](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md#versioning) and export the chain from the `index.ts` file

**Comment by msantamaria86** [comment]
changeset created

**Comment by msantamaria86** [comment]
changes addressed

File: .changeset/fluffy-flowers-hear.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/chains': minor
```
Comment (tmm) [inline .changeset/fluffy-flowers-hear.md:2]:
```suggestion
'@wagmi/chains': patch
```

File: .changeset/real-hairs-exercise.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
+'@wagmi/connectors': minor
+'@wagmi/chains': minor
+---
+
+Added ASTAR's shibuya testnet
```
Comment (tmm) [inline .changeset/real-hairs-exercise.md:6]: Please remove this changeset

File: packages/connectors/src/index.ts [hunk 6]
```diff
@@ -1,3 +1,26 @@
+import { Chain } from '../../chains/src'
 export { Connector } from './base'
 export type { ConnectorData, ConnectorEvents } from './base'
 export type { Ethereum } from './types'
+
+export const shibuya = {
```
Comment (tmm) [inline packages/connectors/src/index.ts:6]: Please remove this change. `shibuya` should be exported from `packages/chains/src/index.ts`

File: packages/chains/src/shibuya.ts [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    public: { http: ['https://evm.shibuya.astar.network/'] },
+  },
+  blockExplorers: {
+    default: {
+      name: 'Shibuya Explorer',
+      url: 'https://shibuya.subscan.io',
+    },
+  },
+  testnet: true,
+}
```
Comment (tmm) [inline packages/chains/src/shibuya.ts]:
```suggestion
import { Chain } from './types'

export const shibuya = {
  id: 81,
  name: 'Shibuya Network',
  network: 'Shibuya-network',
  nativeCurrency: {
    decimals: 18,
    name: 'shibuya',
    symbol: 'SBY',
  },
  rpcUrls: {
    default: { http: ['https://evm.shibuya.astar.network/'] },
    public: { http: ['https://evm.shibuya.astar.network/'] },
  },
  blockExplorers: {
    default: {
      name: 'Shibuya Explorer',
      url: 'https://shibuya.subscan.io',
    },
  },
  testnet: true,
} as const satisfies Chain
```

**Review by tmm — CHANGES_REQUESTED** [review]

**Comment by jxom** [comment]
Didn't mean to close. PR at #172 (commits preserved)

**Pull request closed without merging**